pkg/reconciler/v1alpha1/service/resources/route_test.go

```diff
@@ -28,7 +28,7 @@ func TestRouteRunLatest(t *testing.T) {
 	testConfigName := names.Configuration(s)
 	r, err := MakeRoute(s)
 	if err != nil {
-		t.Errorf("expected nil for err got %q", err)
+		t.Errorf("unexpected error: %v", err)
```

Comment:
You changed most error messages to start with an uppercase. Here as well?

Reply:
Just updated the whole file.